src/js/components/Breadcrumbs.js

```diff
@@ -0,0 +1,242 @@
+import classNames from 'classnames';
+import DeepEqual from 'deep-equal';
```

Comment:
question: do we usually camelcase the things that we import that are functions? like `classNames`